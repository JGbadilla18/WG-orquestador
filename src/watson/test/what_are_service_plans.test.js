const testWatson = require('./testWatson');

watsonText = 'Plan mi apoyo'

it(watsonText, () => {
  expect.assertions(1);
  return testWatson(watsonText).then(data => expect(data).toMatchSnapshot());
});
