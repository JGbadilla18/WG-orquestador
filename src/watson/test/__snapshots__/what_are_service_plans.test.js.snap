// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Plan mi apoyo 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "buttonType": "web_url",
      "buttons": Array [
        Object {
          "title": "Aquí",
          "url": "https://blog.corporacionbi.com/productos/boletin-plan-mi-apoyo-y-plan-mi-comodidad",
        },
      ],
      "response_index": 0,
      "response_type": "button",
      "text": "Los Planes de Servicio Mi Apoyo y Mi Comodidad son una combinación de productos y servicios complementarios a la cuenta monetaria que Banco Industrial te ofrece a un precio menor que si se adquieren por separado. 
Conoce sobre los beneficios de los Planes Bi.",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;
